sources/common/doc/man/vcdMaker.txt

```diff
@@ -1,28 +1,33 @@
 NAME
   vcdMaker - log file to VCD converter
 SYNOPSIS
-  vcdMaker  [-c line-counter-signal-name] -t s|ms|us|ns|ps|fs [-v] -o output-file
+  vcdMaker  [-c line-counter-signal-name] [-u user-log-format] -t s|ms|us|ns|ps|fs [-v] -o output-file
             [--] [--version] [-h] input-file
 DESCRIPTION
   vcdMaker is a tool that converts textual log files to VCD trace files.
 
-  The expected log format is as follows:
+  The inherent vcdMaker log format is as follows:
 
     #Timestamp TopModule.Module1...ModuleN.SignalName SignalValue <signal-size | f | e> Comment
 
-  The first field is a timestamp in some time unit. Please note that log lines need not to be sorted.
+    The first field is a timestamp in some time unit. Please note that log lines need not to be sorted.
 
-  The second field is a signal name. VCD uses hierarchical signal naming, this structure should be reflected in the log line.
+    The second field is a signal name. VCD uses hierarchical signal naming, this structure should be reflected in the log line.
 
-  The third field is a signal value. The value is empty for event signals.
+    The third field is a signal value. The value is empty for event signals.   
 
-  The fourth field is a signal size/type. For integer signals the signal-size should be a signal size in bits, for real number type code 'f' should be used and for event signals type code 'e' should be used.
+    The fourth field is a signal size/type. For integer signals the signal-size should be a signal size in bits, for real number type code 'f' should be used and for event signals type code 'e' should be used.
 
-  The last field is a comment, which is ignored.
+    The last field is a comment, which is ignored.
 
-  The generated log that consists of lines in the above format can be transformed into a VCD trace and later viewed using any graphical tool, eg. GTKWave.
+
+  User defined log format
+    The tool can parse any user log format. Its syntax must be defined by an XML file and passed via the '-u' option. See the full documentation for details.
```

Comment:
Im not really a fan of making "real" documentation and some "worse" man documentation.

Reply:
That wasn't the intention definitely. I have no much experience regarding man documentation. I was always using it to check the basic syntax. For details I was referring somewhere else. But this could be just me. What way shall we go? Would you rather describe the full XML format later in the man? Or shall we provide a link to our wiki?

Reply:
I don't really know. We can just leave it as is for now. :-) If only we could generate both from the same source...